nni/experiment/experiment.py

```diff
@@ -193,8 +207,369 @@ def run(self, port: int = 8080, debug: bool = False) -> bool:
             self.stop()
 
 
-    def get_status(self) -> str:
+    def connect_experiment(self, port: int):
+        """
+        Connect to an existing experiment.
+
+        Parameters
+        ----------
+        port
+            The port of web UI.
+        """
+        self.port = port
+        self.get_status()
+
+
+    def _experiment_rest_get(self, port: int, api: str) -> Any:
```

Comment:
double-line between member functions looks weird to me. Suggest reformatting this file with tools like autopep8.

Reply:
I will confirm with @liuzhe-lz